From 1306d6fa9929809d1ded45d509278ceb7762725a Mon Sep 17 00:00:00 2001
From: Mervin Fansler <mmfansler@gmail.com>
Date: Wed, 14 Jun 2023 23:45:22 -0400
Subject: [PATCH] add makevars

---
 src/Makevars | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 src/Makevars

diff --git a/src/Makevars b/src/Makevars
new file mode 100644
index 00000000..19f08178
--- /dev/null
+++ b/src/Makevars
@@ -0,0 +1 @@
+PKG_CXXFLAGS = -D_LIBCPP_DISABLE_AVAILABILITY
